packages/overlay/src/active-overlay.ts

```diff
@@ -289,10 +313,20 @@ export class ActiveOverlay extends LitElement {
         this.schedulePositionUpdate();
     }
 
-    public render(): TemplateResult {
+    public renderTheme(content: TemplateResult): TemplateResult {
+        import('@spectrum-web-components/theme');
```

Comment:
Don't force non-`sp-theme` users to download this code if they aren't actually using it.